docs/recommendations/RNA/delins.md

```diff
@@ -52,16 +52,7 @@ bin/pull-syntax -c -f docs/syntax.yaml rna.delins
     - **`r.1401_1446delins[NR_002570.3:r.1513_1558]`**<br>
       conversion in exon 9 of the _CYP2D6_ gene, replacing exon 9 nucleotides `r.1401` to `r.1446` with those of the 3' flanking _CYP2D7P1_ gene, nucleotides `r.1513` to `r.1558`.
 
-- Adjoined transcripts from gene fusions (based on [SVD-WG007](../../consultation/SVD-WG007.md)).
-    - **translocation-derived adjoined transcript**<br>
-      `NM_152263.2:r.-115_775::NM_002609.3:r.1580_*1924` describes an adjoined transcript from a `TPM3::PDGFRB` gene fusion, where nucleotides `r.-115` to `r.775` (reference transcript `NM_152263.2`, _TPM3_ gene) are coupled to nucleotides `r.1580` to `r.*1924` (reference transcript `NM_002609.3`, _PDGFRB_ gene).
-
-    - **deletion-derived adjoined transcripts**
-        - **`NM_002354.2:r.-358_555::NM_000251.2:r.212_*279`**<br>
-          describes an adjoined transcript from an `EPCAM::MSH2` gene fusion, where nucleotides `r.-358` to `r.555` (reference transcript `NM_002354.2`, _EPCAM_ gene) are coupled to nucleotides `r.212` to `r.*279` (reference transcript `NM_000251.2`, _MSH2_ gene).
-
-        - **`NM_002354.2:r.?_555::guaugauuuuuuaataa::NM_000251.2:r.212_?`**<br>
-          describes an adjoined transcript from an `EPCAM::MSH2` gene fusion, where only the fusion break point has been characterised, showing the insertion of a 17 nucleotide sequence (`guaugauuuuuuaataa`) between two adjoined transcripts.
+- Example adjoined transcripts from gene fusions (based on [SVD-WG007](../../consultation/SVD-WG007.md)) are discussed in the [adjoined transcript recommendations](adjoined_transcript.md).
```

Comment:
I don't think we need to reference WG007 any longer. Instead, just...

- Example adjoined transcripts from gene fusions are discussed in [Adjoined Transcript](adjoined_transcript.md).

Reply:
I agree; addressed in fed2773